gems/aws-sdk-s3/spec/presigner_spec.rb

```diff
@@ -116,7 +109,22 @@ module S3
             time: Time.utc(1969, 4, 20)
           }
           actual_url = pre.presigned_url(:get_object, params)
-          expect(actual_url).to eq(expected_url)
+          expect(actual_url).to include(date_time)
```

Comment:
Just a side note, not required, I'd prefer a more clear `&X-Amz-Date=19690420T000000Z` match or `&X-Amz-SignedHeaders=host&user-agent`,